### DISCUSSION TITLE
品技(原91TVG) 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#45] [模板讨论]
品技(原91TVG) 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by zhuzhuani** [comment]
flaresolverrUrl 填写什么？
https://www.tekqart.com/
www.tekqart.com
都不行。。

**Reply by wjf0214** [reply]
自建的docker服务地址。官方是`flaresolverr/flaresolverr`镜像,但是目前官方版本有问题。cookie没法传递，截至目前还未更新修复。暂用第三方修改的版本。为`15cm/flaresolverr`容器镜像。
地址就是搭建完成后的访问地址，如【`https://xxx.xxx.com/v1`】的格式。

**Reply by zhuzhuani** [reply]
明白了 十分感谢！

**Reply by zhuzhuani** [reply]
成功啦~十分感谢

**Comment by qhq** [comment]
感谢，甲骨文上官方的架起来了，但签到不行。15cm/flaresolverr的在甲骨文上好像架不起来，提示 `The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

**Reply by wjf0214** [reply]
官方的支持架构`linux/386,linux/amd64,linux/arm/v7,linux/arm64/v8`,但是有问题。
`15cm/flaresolverr`的可能不支持`linux/arm64/v8`，我没仔细看。
目前只能看看有没有构建支持`linux/arm64/v8`的可用镜像，或者等官方修复支持cookie。
又或者看自己能不能尝试更改构建`linux/arm64/v8`的可用镜像了。

**Comment by shallwei** [comment]
感谢大佬更新 但是用不太来。怎么填写cookie？ dooker是自建在unraid上的

**Reply by wjf0214** [reply]
具体可以参考首页的参数说明，如下图
![image](https://user-images.githubusercontent.com/23102821/216210345-a2acfe32-0c59-4bbd-bc51-fe1f36d3300d.png)
flaresolverrUrl 的具体说明可以参考上面的解释。
因为这个网站有cf盾，所以要利用其他工具来过盾。而且因为目前这个工具还不是特别成熟，还有些问题，所以麻烦点。

**Reply by shallwei** [reply]
感谢大佬回复,今天熟悉了下flaresolverr，已经成功，踩了个坑就是 不熟 url 要填 http://xxxxx.com/v1 中这个/v1一定要

**Comment by xuanxuaner** [comment]
打开 http://xxxxx.com/v1 提示{"error": "Method not allowed.", "status_code": 405}是不是没有设置正确？

**Reply by hecarli555** [reply]
同问。。。我也是这问题

**Reply by cai23511** [reply]
请问这个问题解决了吗，什么方法呢

**Comment by Yesinger** [comment]
大家的签到还正常吗？
91tvg改名tekqart回来以后，我把原来那个91tvg模板里的91tvg全替换成tekqart以后是可以签到的，持续了1个月都正常。
但是直到2月26号以后就再也没成功了，更新cookie也不起作用。
所以刚刚我找到了这个模板，配置了flaresolverr，flaresolverrUrl栏填的是http://{dokcer所在主机ip}:8191/v1，flaresolverr的docker日志里是可以看到post记录的，并且确认XnMp_2132_auth、XnMp_2132_saltkey均无误，但是依然签到失败，不知道是什么原因，特来请教。

**Comment by shallwei** [comment]
我今天也发现失效了 签到了一个月正常，换了cookie， flaresloverr 日志也看到记录。有相同情况没
日志：Failed at 1/10 request,
Fail assert: {"re": "\"status\": \"error\"", "from": "content"} from failed_asserts,
Request URL: {{flaresolverrUrl}}

**Reply by dj1234618** [reply]
一样的问题。。

**Reply by SeanNaive** [reply]
同样的问题，你们解决了没

**Reply by shallwei** [reply]
等作者修复了，感觉是flaresolver出问题了

**Reply by wjf0214** [reply]
@dj1234618 @zhangsxnaive @shallwei 
我刚更新了一版，可以试试。
2个cookie参数改为一个完整的cookie，另外改用官方版本的flaresolver了。

**Reply by shallwei** [reply]
改用官方版本的flaresolver  +最新的一版 已经可以签到了，好像官方的不用https，大家注意下 http即可。再次感谢大佬更新

**Reply by Nighting0518** [reply]
> 改用官方版本的flaresolver +最新的一版 已经可以签到了，好像官方的不用https，大家注意下 http即可。再次感谢大佬更新


大佬
我更换了最新官方版本的flaresolver和最新的模板 但是签到还是提示

2023-03-23 22:38:58
日志：Failed at 2/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

这个https是要修改模板里面所有的https为http么？

**Reply by SeanNaive** [reply]
> > 改用官方版本的flaresolver +最新的一版 已经可以签到了，好像官方的不用https，大家注意下 http即可。再次感谢大佬更新
> 
> 大佬 我更换了最新官方版本的flaresolver和最新的模板 但是签到还是提示
> 
> 2023-03-23 22:38:58 日志：Failed at 2/12 request, Fail assert: {"re": "200", "from": "status"} from success_asserts, Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received, Request URL: {{flaresolverrUrl}}
> 
> 这个https是要修改模板里面所有的https为http么？

是修改flaresolver为http，比如http://192.168.1.1:8191/v1

**Comment by huoshenw** [comment]
日志：Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}
用的是官方最新版flaresolver 
Incoming request => POST /v1 body: {'cmd': 'request.get', 'url': 'https://www.tekqart.com/plugin.php?id=dc_signin', 'maxTimeout': 60000}
2023-06-17 13:03:09 INFO     Challenge detected. Title found: Just a moment...
2023-06-17 13:03:43 INFO     Challenge solved!

**Reply by wjf0214** [reply]
这是请求超时，手动有可能过cf盾的时候会超时，添加完成后可以等自动的看看。没问题就OK。

**Reply by hecarli555** [reply]
这几天我也是超时了。。。品技域名都优选ip还是不行，无论自动还是手动都是超时。。。怎么解决？

**Reply by wjf0214** [reply]
flaresolverr 好像暂时不支持过cf这个盾。可能要等更新了。

**Reply by hecarli555** [reply]
flaresolverr这玩意不就是用来过cf盾的么。。。。

**Reply by wjf0214** [reply]
是这样，但是cf盾更新了，过盾的也得等一阵才能适配，这样的道理。不太可能cf盾一更新，过盾的同步就能用。等等吧
我看有人已经反馈了

**Reply by hecarli555** [reply]
哦哦哦懂了，，，原来是cf更新了，那明白了，那只能等作者更新了。
不过也奇怪，我国内另一个朋友，环境也是电信的，他的就穿盾正常能签到，我的就不行。。。。

**Reply by wjf0214** [reply]
网站这两天好像给盾去掉了，暂时可以用了。

**Reply by hecarli555** [reply]
我也发现了，这是品技主动把盾撤掉了？

**Comment by jerrell851** [comment]
@wjf0214 请问大佬，这个HAR里面的空间访问次数是个啥？我目前能签到成功，空间访问次数我填的1

**Reply by wjf0214** [reply]
访问别人主页的次数。奖励可以看积分规则，一般10次。

**Reply by jerrell851** [reply]
谢谢。明白了。积分规则里每天最多9次。所以我填9。

**Comment by xuexijiaoben** [comment]
还是提示这个错误，折腾好几次了
‘Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30002 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

用的官网的
msg | "FlareSolverr is ready!"
version | "3.3.2"
userAgent | "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36"

**Comment by qhq** [comment]
现在没盾了，有没有没盾的 签到版本呀

**Comment by irvineyang** [comment]
提示错误为：
forbidden opcode(s) in 'cf_clearance': RESUME

用得官网的
{"msg": "FlareSolverr is ready!", "version": "3.3.4", "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"}

请教如何解决。谢谢

**Reply by irvineyang** [reply]
解决了，是qiandao的框架不够新。更新最新的就搞定了

**Comment by irvineyang** [comment]
qiandao的框架是最新的了：230821

flaresolver用的官方的
{"msg": "FlareSolverr is ready!", "version": "3.3.4", "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"}

昨天签到成功一次。。
今天就失败了，有两种提示：

第一种
Failed at 4/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.tekqart.com/

第二种
Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

flaresolverrUrl填写的是 **http://192.168.1.4:8191/v1**
请教这个咋解决

**Reply by wjf0214** [reply]
第一种感觉是参数没获取正确，不知道是不是还有其他特殊异常没捕获到。第二种就是过盾的请求超时了。如果定时执行没问题的话不用管。

**Reply by irvineyang** [reply]
今天还没签到成功过呢~
第一次是自动签到 第二种错误
第二次是手动执行 第二种错误
第三次是手动执行 第一种错误
第四次是手动执行 第一种错误
第五次是手动执行 第二种错误
第六次是手动执行 弟一种错误

六次均失败，两种错误类型各占一半。
昨天成功的CK今天也是无法使用，中间重新抓取过CK也不成功。

**Reply by irvineyang** [reply]
![微信截图_20230913152256](https://github.com/wjf0214/qd-templates/assets/61082210/546dc769-884b-42f3-87b2-23990e5e9349)
我看到编辑模版里面的预览，有显示“Enable JavaScript and cookies to continue”，不知道是不是有关。

**Comment by jianjam** [comment]
用的官方的 FlareSolverr 签到框架是20230821，都是在本地docker上的，运行还是报错
Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Connection timed out after 30000 milliseconds,
Request URL: {{flaresolverrUrl}}。
这样要如何解决啊

**Reply by irvineyang** [reply]
检查你的docker上的网络是否正常。
我今天也发现599错误了，然后发现是梯子出问题。。

**Comment by ngankalok** [comment]
我今天也发现失效了

2023-10-17 00:54:01
日志：Failed at 1/3 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.tekqart.com/plugin.php?id=dc_signin

**Reply by wjf0214** [reply]
看你这提示，模板都不对吧？

**Reply by ngankalok** [reply]
应该是，我重新弄一个吧，有没有一个好点的教程~我不太熟~~谢谢大神

**Comment by irvineyang** [comment]
我这两天也一直报错了。

Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

**Comment by jerrell851** [comment]
我的从10月15日开始报错，错误提示也是

Failed at 2/16 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,Request URL: {{flaresolverrUrl}}

**Comment by hecarli555** [comment]
也是一样开始报错
2023-10-21 16:01:44 
日志：Failed at 2/16 request, 
Fail assert: {"re": "200", "from": "status"} from success_asserts, 
Response Error : HTTP 599: Operation timed out after 30002 milliseconds with 0 bytes received, 
Request URL: {{flaresolverrUrl}}

**Comment by hecarli555** [comment]
大佬不修复一下么？

**Reply by wjf0214** [reply]
正常的，这个错误是 flaresolverr 过盾超时了，要不重试几次，要不把超时设置调长。

**Reply by hecarli555** [reply]
超时设置是怎么弄来着。。。？有步骤么？

**Comment by irvineyang** [comment]
模板中第二条Request Payload中

{
    "cmd": "request.get",
    "url":"https://www.tekqart.com/plugin.php?id=dc_signin",
    "maxTimeout": 60000
}

我把maxTimeout的值从60000改成600000，也还是1分钟报错。
是修改这里吗？
怎么我修改成10倍后还是1分钟就报错。。

**Reply by wjf0214** [reply]
这个是单指这个请求的超时时间，影响提示报错的还有，flaresolverr 反代设置的超时时间，框架默认的请求超时时间。1分钟应该是flaresolverr 的超时时间。框架我记得是半分钟。

**Reply by hecarli555** [reply]
大佬您说的那些修改内容能不能直接模板上修改更新一下嘛。。。萌新都不知道在哪里改。。。。

**Reply by wjf0214** [reply]
和模板关系不大，是本地环境配置问题。而且一般来说不用管就行，我这一般失败重试2,3最多就成功了。就是可能错误提示多点。

**Reply by irvineyang** [reply]
我这边默认的配置10+次都不行。。都还是599错误，已经没有成功过了。

我重新删了flaresolverr的容器，重新建，加上了browser_timeout参数，设定为180000毫秒，但是实测下来还是40秒就报599错误了。。

docker run -d --name=flaresolverr -p 8191:8191 -e LOG_LEVEL=info -e BROWSER_TIMEOUT=180000 --restart unless-stopped flaresolverr/flaresolverr:latest

**Comment by hecarli555** [comment]
今天又开始报错了，。。。。
<html>
<body>

Failed at 2/16 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 599: Operation timed out after 30002 milliseconds with 0 bytes received,Request URL: {{flaresolverrUrl}}
--



</body>
</html>

**Reply by shallwei** [reply]
一样的问题
Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}
查看了下flaresolverr 
日志显示Internal Server Error

**Reply by wjf0214** [reply]
cf升级了，过盾失败，暂时官方的版本不支持，只能等升级解决。不过我看有人处理了，你们可以临时用下测试看看， docker镜像为`21hsmw/flaresolverr:test` 来过盾试试。

**Reply by hecarli555** [reply]
大佬牛逼！亲测成功了！安全性应该没问题吧这个第三方？

**Reply by shallwei** [reply]
官方刚更新了[v3.3.12]，亲测已经可以了

**Reply by a502347020** [reply]
确实之前一直失败，更新flaresolverr 3.3.12，签到正常

**Comment by xuexijiaoben** [comment]
Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30001 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

更新上面说的最新官方版，还是这样，晕呀

**Reply by hecarli555** [reply]
试试第三方

**Comment by xuexijiaoben** [comment]
> 试试第三方

试了21hsmw，都不行，一样的错误，
看日志有以下错误
version_main cannot be converted to an integer
Error: Error solving the challenge. Timeout after 60.0 seconds.
500 Internal Server Error

估计是开着科学或IPV6的原因？

**Reply by hecarli555** [reply]
今天又又又失败了

**Comment by jjjyyyllll** [comment]
似乎过了cf盾，但是还是403错误。用的本地ip做的代理，但是本地ip访问网站一切正常。手动签到一切ok

Failed at 4/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.tekqart.com/

**Reply by hecarli555** [reply]
更新过盾工具

**Reply by jjjyyyllll** [reply]
盾是官方的今天最新版了。。。

**Comment by Fanghome1987** [comment]
2024-03-07 22:26:15
日志：Failed at 2/16 request,
Fail assert: {"re": "\"status\": \"error\"", "from": "content"} from failed_asserts,
Response Error : HTTP 500: Internal Server Error,
Request URL: {{flaresolverrUrl}}

我怎么出的问题和大家都不一样。。。

**Reply by IV16SL** [reply]
一样。

**Reply by wjf0214** [reply]
这是过盾失败了。flaresolverr是不是最新的？

**Reply by Fanghome1987** [reply]
是官方最新的版本

**Reply by wjf0214** [reply]
参数都没填写错误的情况下，你按顺序手动执行下看看过盾失败报错提示的什么。

**Comment by Nighting0518** [comment]
更新最新版本flaresolverr后依旧报错
2024-03-22 10:33:07
日志：Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

flaresolverr日志显示
![image](https://github.com/wjf0214/qd-templates/assets/26594867/20ddbd4d-93da-4476-ba81-6c1b8ba9c765)

根据信息搜索到了这篇文章https://www.zenrows.com/blog/flaresolverr#flaresolverr-alternative
怀疑报错是否因为flaresolverr无法绕过最新的Cloudflare

**Reply by hecarli555** [reply]
你猜对了，等过盾工具更新了只能

**Comment by shallwei** [comment]
更新昨天release最新版本flaresolverr v3.3.17后依旧报错

**Comment by shallwei** [comment]
三个月了 还是不行 V3.3.21

**Comment by hecarli555** [comment]
三个月了 还是不行 V3.3.21，大佬您还修不修啊？

**Comment by yb9210** [comment]
还是不行
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}

**Reply by hecarli555** [reply]
> 还是不行 Fail assert: {"re": "200", "from": "status"} from success_asserts, Response Error : HTTP 599: Operation timed out after 30000 milliseconds with 0 bytes received, Request URL: {{flaresolverrUrl}}

正常啊，参考真白萌的签到模板方案

**Comment by hecarli555** [comment]
今天莫名其妙失败了
Failed at 2/16 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Operation timed out after 30004 milliseconds with 0 bytes received,
Request URL: {{flaresolverrUrl}}
